legal-api/src/legal_api/resources/v2/business/business_documents.py

```diff
@@ -76,41 +77,56 @@ def _get_document_list(business):
         documents['documents'][doc] = f'{base_url}{doc_url}/{doc}'
 
     if business.legal_type == Business.LegalTypes.COOP.value:
-        coop_documents = _get_coop_documents_list(business)
-        for coop_doc_key, coop_doc_value in coop_documents.items():
-            documents['documents'][coop_doc_key] = coop_doc_value
+        documents['documentsInfo'] = {}
+        coop_documents, coop_info = _get_coop_documents_and_info(business)
+        for k, v in coop_documents.items():
+            documents['documents'][k] = v
+        for k, v in coop_info.items():
+            documents['documentsInfo'][k] = v
 
     return jsonify(documents), HTTPStatus.OK
 
 
-def _get_coop_documents_list(business):
-    """Get certified memorandum and rules for coop."""
-    coop_documents = {}
+def _get_coop_documents_and_info(business):
+    """Get certified memorandum and rules documents + info for coop."""
+    documents, info = {}, {}
 
     if not business:
-        return coop_documents
+        return documents, info
 
     base_url = current_app.config.get('LEGAL_API_BASE_URL')
     base_url = base_url[:base_url.find('/api')]
     business_id = business.id
     business_identifier = business.identifier
 
-    coop_rules_document = Document.find_by_business_id_and_type(business_id, DocumentType.COOP_RULES.value)
-
-    if coop_rules_document:
-        coop_rules_filing = Filing.find_by_id(coop_rules_document.filing_id)
-        coop_rules_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
-                                                              'filing_id': coop_rules_filing.id,
+    rules_document = Document.find_by_business_id_and_type(business_id, DocumentType.COOP_RULES.value)
+
+    if rules_document:
+        rules_filing = Filing.find_by_id(rules_document.filing_id)
+        rules_doc_url = url_for('API2.get_documents', **{'identifier': business_identifier,
+                                                         'filing_id': rules_filing.id,
+                                                         'legal_filing_name': None})
+        documents['certifiedRules'] = f'{base_url}{rules_doc_url}/certifiedRules'
+        info['certifiedRules'] = {
+            'key': rules_document.file_key,
+            'name': DocumentMetaService.general_filename(
+                rules_filing.filing_date.isoformat(), business.identifier, 'Certified Rules'),
```

Comment:
@vysakh-menon-aot let me know if this is ok

Reply:
I think `filing_date` is the right one since we certify and replace the file while processing the filing

Reply:
Sounds good